server.js

```diff
@@ -46,7 +47,11 @@ if (fs.existsSync(legacySecretsPath)) {
 }
 
 const Server = require('./core/server/server')
-const server = (module.exports = new Server(config))
+const server = (module.exports = new Server(config, {
+  id: process.env.INSTANCE_ID,
+  env: process.env.INSTANCE_ENV || process.env.NODE_CONFIG_ENV || 'unknown',
```

Comment:
@paulmelnikow would you like to handle `env` similarly to `instanceIdFrom`? I think we can simply use https://github.com/badges/shields/blob/master/config/custom-environment-variables.yml for reading `env`. 

If we want to remove `env` from the piece of code above, I suggest to remove instance metadata object.

Reply:
Hmm, to keep things simple I'd suggest we do one of these two:

1. Always use NODE_CONFIG_ENV
2. Specify the influx env in config as a string literal, and rely on setting it using the env var set in custom_environment_variables

Reply:
I chose option 2, because this will work for review apps - we want to set the env value to the PR review application name.